src/ContextProvider/index.js

```diff
@@ -5,28 +5,28 @@ import { Component, PropTypes, Children } from "react"
 export default class StatinamicContext extends Component {
   props: {
     collection: StatinamicCollection,
-    layouts: Object,
+    layouts: Object, // deprecated
```

Comment:
You are right. But without collection or layout, we will have issue. Will check this before removing proptypes